api/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "API for the Semantic Scholar Reader",
   "main": "index.js",
   "scripts": {
-    "start": "ts-node src/index.ts",
+    "start": "ts-node-dev src/index.ts",
```

Comment:
The `ts-node-dev` utility watches files for changes and automatically restarts the service whenever they occur.

A nice little utility for rapid feedback :).

Reply:
Nope! Good catch.

Reply:
Removed!